queue/src/fiskaltrust.Middleware.Queue/SignProcessor.cs

```diff
@@ -178,7 +185,7 @@ private async Task<ReceiptResponse> InternalSign(ftQueue queue, ReceiptRequest d
                                 Data = e.ToString()
                             }
                         },
-                        ftState = (long) (((ulong) data.ftReceiptCase & 0xFFFF_0000_0000_0000) | 0x2000_EEEE_EEEE)
+                        ftState = (long)(((ulong)data.ftReceiptCase & 0xFFFF_0000_0000_0000) | 0x2000_EEEE_EEEE)
                     };
                 }
```

Comment:
Should we create activity traces here too?

Reply:
you mean something like this?

`Activity.Current?.Source.StartActivity("InternalSign")`?